graphql/schemabuilder/pagination.go

```diff
@@ -135,6 +150,11 @@ func (funcCtx *funcContext) constructConnType(sb *schemaBuilder, typ reflect.Typ
 
 	pageInfoType, _ := reflect.TypeOf(Connection{}).FieldByName("PageInfo")
 	pageInfoField, err := sb.buildField(pageInfoType)
+	pageInfoNonNull, _ := pageInfoField.Type.(*graphql.NonNull)
+	pageInfoObj := pageInfoNonNull.Type.(*graphql.Object)
+	if returnsConnInfo {
```

Comment:
can you add a comment for why decided on this?

Reply:
done